modules/installation-about-custom-gcp-vpc.adoc

```diff
@@ -32,6 +32,15 @@ To ensure that the subnets that you provide are suitable, the installation progr
 * The subnet CIDRs belong to the machine CIDR.
 * You must provide a subnet to deploy the cluster control plane and compute machines to. You can use the same subnet for both machine types.
 
+If you are working in a disconnected environment, you are unable to reach the
+public IPs for EC2 and ELB endpoints. To resolve this, you must create a VPC
+endpoint and attach it to the subnet that the clusters are using. The endpoints
+should be named as follows:
+
+* `ec2.<region>.amazonaws.com`
```

Comment:
this wouldn't apply to GCP, it is AWS specific.  I'm not aware of how it works in GCP, maybe @sdodson can help us out.

Reply:
Yeah, this is not necessary for GCP. Lets drop this change.

Reply:
@pweil- @sdodson Thanks! Updated.